tests/queries/0_stateless/03541_table_without_insertable_columns.sql

```diff
@@ -0,0 +1,12 @@
+-- Tags: no-parallel, no-ordinary-database
+-- Tag no-parallel: static UUID
```

Comment:
You can use `no-flaky-check` instead. The test itself is parallel with any other test, just not with itself.